app/forms/thredded/private_topic_form.rb

```diff
@@ -82,7 +82,7 @@ def private_user_ids
     def normalized_user_ids
       user_ids
         .reject(&:empty?)
-        .map(&:to_i)
+        .map(&:to_s)
         .push(user.id)
```

Comment:
This needs to be converted `to_s` as well, otherwise the `uniq` below won't work.